### DISCUSSION TITLE
Support for creating a torrent containing Web Seeds

### DISCUSSION THREAD

**Discussion by cherinyy** [transmission/transmission#4845] [Ideas]
Hello,

It seems that the current transmission does not support the creation of Torrents containing Web Seeds, what is the reason for this decision? I think that the use of Web Seeds + P2P BitTorrent networks helps to reduce the pressure on some websites' download servers, which is a very practical way to use them.

Best regards,
wordlesswind

**Comment by ckerr** [comment]
Sounds like a good idea. If you'd like to submit a patch I'd be happy to do review.
